go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/text v0.3.7
 	golang.org/x/time v0.0.0-20220411224347-583f2d630306
-	gopkg.in/ini.v1 v1.62.0
+	gopkg.in/ini.v1 v1.66.4
```

Comment:
conflicts on the go.mod.

These updates seem fine, but curious if they was a specific motivation for the bumps?

Reply:
Oh, nvm no conflicts.  I had an old branch, sorry.